### ISSUE TITLE
Map volunteer filter

### ISSUE THREAD

**Issue by unrevised6419** [code4moldova/covid19md-voluntari-client#150]
https://www.figma.com/file/oCN3NSECKQnS4PWhZnWGCT/Dashboard_Voluntar.md_V2.0?node-id=613%3A1949

do after #149

**Comment by unrevised6419** [comment]
Closing in favor of #151
